### PR TITLE
feat: support RTL in components

### PR DESCRIPTION
**Pull request by amje**

### CODE REVIEW

File: src/components/Button/ButtonIcon.tsx [hunk 1]
```diff
@@ -6,10 +6,17 @@ const b = block('button');
 
 type Props = React.PropsWithChildren<{
     className?: string;
-    side?: 'start' | 'end';
+    side?: 'left' | 'right' | 'start' | 'end';
 }>;
 
 export const ButtonIcon = ({side, className, children}: Props) => {
+    if (side === 'left') {
```
Comment (korvin89) [inline src/components/Button/ButtonIcon.tsx:13]: Lets add a new var here?

File: src/components/Popup/Popup.scss [hunk 2]
```diff
@@ -75,15 +75,17 @@ $transition-distance: 10px;
     }
 
     &[data-popper-placement*='left'] #{$block}__arrow {
-        inset-inline-end: -$arrow-offset;
+        // stylelint-disable-next-line csstools/use-logical
```
Comment (korvin89) [inline src/components/Popup/Popup.scss:78]: Should we add todo here?

Comment (amje) [reply src/components/Popup/Popup.scss:78]: No, popper.js uses physical properties, it doesn't support RTL, we should migrate to Floating UI, which does

File: src/components/Button/ButtonIcon.tsx (hunk 1 above)
Comment (amje) [reply src/components/Button/ButtonIcon.tsx]: done

**Review by korvin89 — APPROVED** [review]

**Pull request merged**